settings.gradle

```diff
@@ -21,3 +21,4 @@ dependencyResolutionManagement {
         }
     }
 }
+include ':stripe-ui-core'
```

Comment:
nit: please align this with other includes

Reply:
Done